api/doris/v1/types.go

```diff
@@ -281,6 +285,20 @@ type MountConfigMapInfo struct {
 	MountPath string `json:"mountPath,omitempty"`
 }
 
+type Secret struct {
+	// name of secret that needs to mount.
+	SecretName string `json:"secretName,omitempty"`
+
+	// Current Secret Mount Path, default is "/etc/doris"
+	// +optional
+	MountPath string `json:"mountPath,omitempty"`
+
+	// Mounted read-only if true, read-write otherwise (false or unspecified).
+	// Defaults to false.
+	// +optional
+	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,2,opt,name=readOnly"`
```

Comment:
1. There is no need to provide readonly configuration here, its default policy is false, we need this configuration to allow users to modify it midway
2. This attribute of protobuf does not need to be added